js/blog.js

```diff
@@ -0,0 +1,56 @@
+//getting section for showing blog data
+const section = document.querySelector("#blog-text");
+let html = ``;
+//template
+const addTemplate = (post) => {
+  let { body, title, author } = post;
+  let postTitle = title.length > 33 ? `${title.substring(0, 30)}...` : title;
+  // let description = body.length > 30 ? `${body.substring(0, 50)}...` : body; //body truncate
```

Comment:
?

Reply:
For the **...** which appears if the title is too long.